src/Components/Paginator/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+import styles from './style.module.scss';
+
+function Paginator(props) {
+  // Need to pass in number of entries and desired entries per page
+  const { entries, entriesPerPage } = props;
+  const pages = Math.ceil(entries / entriesPerPage);
+  const pageNumbers = [];
+  const [currentPage, setCurrentPage] = useState(1);
+
+  // Below 3 lines can be reused on the page component
+  // const indexOfLastEntry = currentPage * entriesPerPage;
+  // const indexOfFirstEntry = indexOfLastEntry - entriesPerPage;
+  // const currentEntries = entries.slice(indexOfFirstEntry, indexOfLastEntry);
+
+  // Get pages
+  for (let i = 1; i <= Math.ceil(pages); i += 1) {
```

Comment:
I dont think you need `Math.ceil` here again looks like you did it already on line 8

Reply:
✅